### PR TITLE
chore: data & monitor access endpoints to use RTK

### PR DESCRIPTION
**Pull request by manomage**

### CODE REVIEW

File: src/pages/monitor-access/components/BoughtDataTable.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ import { useAppSelector } from '@shared/hooks/useStore';
 import { useAuth } from '@contexts/auth-provider';
 import CustomButton from '@components/ui/Button';
 import dayjs from 'dayjs';
+import { useGetOutgoingPermissionsQuery } from '@store/monitor-access/api';
```
Comment (MHHukiewitz) [inline src/pages/monitor-access/components/BoughtDataTable.tsx:8]: Not used import?

File: src/pages/monitor-access/components/SoldDataTable.tsx [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Starred } from '@components/form';
 import { Link } from 'react-router-dom';
 import CustomTable, { ITableColumns } from '@components/ui/CustomTable';
+import { useGetOutgoingPermissionsQuery } from '@store/monitor-access/api';
```
Comment (MHHukiewitz) [inline src/pages/monitor-access/components/SoldDataTable.tsx:4]: Again, not used import?

**Review by MHHukiewitz — DISMISSED** [review]
Minor notes, otherwise LGTM

File: src/pages/monitor-access/components/BoughtDataTable.tsx (hunk 1 above)
Comment (manomage) [reply src/pages/monitor-access/components/BoughtDataTable.tsx]: Done

File: src/pages/monitor-access/components/SoldDataTable.tsx (hunk 2 above)
Comment (manomage) [reply src/pages/monitor-access/components/SoldDataTable.tsx]: Done

**Review by MHHukiewitz — APPROVED** [review]

**Pull request merged**